web/assets/dropdown-menu.js

```diff
@@ -0,0 +1,15 @@
+export default function dropdownMenu() {
+  document.querySelectorAll("[data-module='dropdown-menu']").forEach((e) => {
+    let dropdownMenuToggle = e.querySelector(
+      "[data-role='dropdown-menu-toggle']",
+    );
+
+    if (dropdownMenuToggle === null) {
+      return;
+    }
+
+    dropdownMenuToggle.addEventListener("change", (e) => {
+      window.location.href = encodeURI(e.target.value);
```

Comment:
I think it would be good to add a bit of defence around the URL to avoid hijacks. It should always start with `/` but not with `//`
```suggestion
      const url = encodeURI(e.target.value);
      if (url.startsWith("//") || !url.startsWith("/")) {
        return;
      }
      
      window.location.href = url
```